lib/util/http/gerrit.ts

```diff
@@ -0,0 +1,48 @@
+import is from '@sindresorhus/is';
+import JSON5 from 'json5';
+import type { HttpOptions, HttpResponse, InternalHttpOptions } from './types';
+import { Http } from './index';
+
+let baseUrl: string;
+export const setBaseUrl = (url: string): void => {
+  baseUrl = url;
+};
+
+/**
+ * Access Gerrit REST-API and strip-of the "magic prefix" from responses.
+ * @see https://gerrit-review.googlesource.com/Documentation/rest-api.html
+ */
+export class GerritHttp extends Http {
+  magicPrefix = /^\)]}'\n/g;
```

Comment:
```suggestion
  private static magicPrefix = /^\)]}'\n/g;
```
should use our regex helper